### ISSUE TITLE
Switch to open license

### ISSUE THREAD

**Issue by Ainali** [publiccodenet/blog#9]
Is there any reason for the blog not to be CC 0 as well?

**Comment by Ainali** [comment]
Images could have other licenses. Wikipedia for example has the text under CC BY-SA and images can be any free license. If we don't want them remixed (but why wouldn't we?) we could have images under CC BY-NC-ND (which is almost useless for reusing). 
Also note that a CC license does not imply that other personality right laws can be disregarded by reusers.

**Comment by ElenaFdR** [comment]
This is all true. The UK government broadly uses CC BY 2.0 or CC BY-NC 2.0 for its photography; there were no issues with inappropriate image reuse in my time there.

**Comment by Ainali** [comment]
So, can we change this? I suggest that if we feel a little bit insecure that we go with CC BY-SA 4.0 for everything since that is a quite strict license (but still free). If we don't think there will be any issue we should go CC 0 for everything. Either way I would be happy. Any thoughts from @bvhme?
